src/hermes/model/context_manager.py

```diff
@@ -54,11 +54,21 @@ def __init__(self, project_dir: pathlib.Path = pathlib.Path.cwd()):
     def prepare_step(self, step: str, *depends: str) -> None:
         self._current_step.append(step)
 
+    ''' @FIXME #373:
```

Comment:
```suggestion
    '''
    FIXME: Implement this to fix #373
```

@SKernchen re your question: This is how I imagined we mark things we come across that need to be fixed during the next refactoring cycle, but not right now. 🤷

In the end, the style doesn't matter, as long as it's somewhat consistent. I usually use plain `FIXME` which gets highlighted in all IDEs I use, but we can use `@FIXME` if you prefer...